src/background/index.ts

```diff
@@ -5,6 +5,55 @@ chrome.runtime.onInstalled.addListener((): void => {
   console.log('Extension installed')
 })
 
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if (request.action === 'login') {
+    const clientId = '34932676923-luth1cmkkhoqhq3kaoealqj2poosaln4.apps.googleusercontent.com';
```

Comment:
This has to be an variable which we can change by environment. 

When we release the extension, we will probably have differernt client ID and we might will have a staging environment.